src/ui/views/LandingPages/Welcome/AddressImport/index.tsx

```diff
@@ -0,0 +1,126 @@
+import { Snackbar, Alert, Box } from '@mui/material';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { AllSet, PickUsername, LandingComponents } from '@/ui/FRWComponent/LandingPages';
+import { PageSlider, useNavigation } from '@/ui/utils/landingPage';
+import { storage } from 'background/webapi';
+import { LLPinAlert } from 'ui/FRWComponent';
+import { useWallet } from 'ui/utils';
+
+import GoogleBackup from './GoogleBackup';
+import ImportPager from './ImportPager';
+import RecoverPassword from './RecoverPassword';
+import SetPassword from './SetPassword';
+
+const AddressImport = () => {
```

Comment:
Think this should just be called Import and the folder renamed as such

Reply:
I think calling it address import is to differentiate from the import statement, also there could be other kind of import in the future.

Reply:
It makes sense to differentiate. Are we importing a wallet, profile, or account? I don't think we'd refer to it as an address. It would be good to unify naming